.shell/runBiDiServer.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# Get location of the script.
+SCRIPT=$(readlink -f $0)
```

Comment:
`readlink -f` doesn't work on macOS, you get the error "readlink: illegal option -- f"

Just `cd "$(dirname $0)/.."` will work here I think. (Comment will need tweaking.)

Reply:
done